ansible/roles_ocp_workloads/ocp4_workload_windows_node/tasks/workload.yml

```diff
@@ -0,0 +1,147 @@
+---
+# Implement your Workload deployment tasks here
+- name: Setting up workload for user
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+# Download SSH proxy container creator script
+- name: Download SSH proxy container creator script
+  get_url:
+    url: https://raw.githubusercontent.com/eparis/ssh-bastion/master/deploy/deploy.sh
+    dest: /home/ec2-user/windows_node_scripts/deploy-sshproxy.sh
+    owner: ec2-user
+    group: ec2-user
+    mode: '0555'
+
+# Copy over ssh command script
+- name: Download SSH proxy container creator script
+  copy:
+    src: sshcmd.sh
+    dest: /home/ec2-user/windows_node_scripts/sshcmd.sh
+    owner: ec2-user
+    group: ec2-user
+    mode: '0555'
+
+# Get the OpenShift cluster ID
+- name: Get the OpenShift cluster ID
+  shell: oc get -o jsonpath='{.status.infrastructureName}' infrastructure cluster
+  register: cluster_id
+
+# Get the region for the cluster
+- name: Get the region for the cluster
+  shell: oc get machineset -n openshift-machine-api -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.placement.region}'
+  register: cluster_region
+
+# Get the first availability zone for the cluster
+- name: Get the first availability zone for the cluster
+  shell: oc get machineset -n openshift-machine-api -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.placement.availabilityZone}'
+  register: cluster_az
+
+# Get the latest Windows Server 2019 with Containers AMI
+- name: Get the latest Windows Server 2019 with Containers AMI
+  shell: >
+    aws ec2 describe-images --region={{ cluster_region.stdout }} --filters 'Name=name,Values=Windows_Server-2019*English*Full*Containers*' 'Name=is-public,Values=true' --query 'reverse(sort_by(Images, &CreationDate))[*].{name: Name, id: ImageId}' | jq -r '.[0].id'
```

Comment:
`ec2_instance`

Reply:
@wkulhanek Same error as above

```
TASK [Extract OpenShift machineSet region] *******************************************************************************************************
fatal: [bastion.lax-9248.sandbox1854.opentlc.com]: FAILED! => {"changed": false, "msg": "Failed to import the required Python library (botocore or boto3) on bastion.lax-9248.internal's Python /opt/virtualenvs/k8s/bin/python. Please read module documentation and install in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"
```

Reply:
Seems we need to add boto3 to the virtualenv. I'll add that to the cluster for future use.